requirements.txt

```diff
@@ -1,12 +1,12 @@
-Django==4.2.3
+Django>=3.0,<4.0
```

Comment:
Django should be kept at version 4.2 for this change, shouldn't it? We're just adding a docker setup, not messing with Django.